application/clicommands/HealthCommand.php

```diff
@@ -37,9 +38,21 @@ public function checkAction()
             $health->setDbResourceName($name);
         }
 
+        if ($this->params->get('name')){
+            $check_name = $this->params->get('name');
+        } else { 
+			$check_name = null; 
+		}
+
+
         if ($name = $this->params->get('check')) {
-            $check = $health->getCheck($name);
```

Comment:
The `echo ...` line remains unchanged, so that it can be used after the `if` and does not have to be repeated. Also, you almost never need to use `is_null()`. We actually only use `=== null` or `!== null`. Additionally, I would start `if`s with what makes them special, which here means that something is different if `$checkName` is actually set, i.e.

```php
if ($checkName !== null) {
    $check = $health->getCheck($name, $checkName);
} else {
    $check = $health->getCheck($name);
}

echo ...
exit(...